docs/content/en/docs/configuration/keys.md

```diff
@@ -809,7 +809,7 @@ Configures External Authentication options.
 
 * `<proto>`: can be `http`, `https`, `service` or `svc`.
 * `<name>`: the IP or hostname if `http` or `https`, or the name of a service if `service`. `svc` is an alias to `service`. Note that the hostname is resolved to a list of IP when the ingress is parsed and will not be dynamically updated later if the DNS record changes.
-* `<port>`: the port number, must be provided if a service is used and can be omitted if using `http` or `https`.
+* `<port>`: the port number, must be provided if a service is used and can be omitted if using `http` or `https`. If the service uses named ports, service's target port name or number should be used instead.
```

Comment:
remove "or number"

Reply:
According to the api description, a port name or number can be used here:

https://github.com/kubernetes/api/blob/0478a3e95231398d8b380dc2a1905972be8ae1d5/core/v1/types.go#L4776-L4785

Maybe we can give some highlight on the correct value to use by rephrasing this to "If the service uses named ports, use the service's `port.targetPort` field value instead."?

Reply:
yes, that would clarify :-)